tests/helper_mpi_tests.py

```diff
@@ -0,0 +1,25 @@
+# The purpose of this helper script is to be able to execute
```

Comment:
Secure development lifecycle requirements do not allow to start the execution of some commands passed to the script without the additional safety checks to prevent the malware execution.

That's why the codefactor check is failing below.

From my understanding, it would be better not to have this helper script, but to modify `test_daal4py_spmd_examples.py` and other similar scripts. I.e. let them use JSON related arguments only at rank==0.

Reply:
Problem is, (a) the pytest plugin works in such a way that it requires being passed arguments in the same call to `pytest`, while the tests require running `pytest` under MPI, so it's not possible to pass different arguments to different MPI ranks within the same CLI call; (b) the file `test_daal4py_spmd_examples.py` is also used by the custom runner here: https://github.com/intel/scikit-learn-intelex/blob/main/tests/run_examples.py

Would like to hear some suggestions about how to better achieve the goal here (make only the first rank save a file, which is achieved by passing arguments to `pytest`) if you could think of others.

Reply:
> Secure development lifecycle requirements do not allow to start the execution of some commands passed to the script without the additional safety checks to prevent the malware execution.

Not sure what's the idea behind it - we have .sh files which run arbitrary commands already.

Reply:
Changed the `os.system` invocation to a `pytest.main` call.